### PR TITLE
Allow switching between visual test themes easily

### PR DESCRIPTION
**Pull request by ashmaroli**
So that one does not need to remember the correct theme names to test..

### CODE REVIEW

File: spec/visual/templates/layout.erb [hunk 1]
```diff
@@ -38,18 +38,164 @@
     #page-footer {
       border-top: 1px dotted
     }
+
+    #theme-selector {
+      display: inline-block;
+      position: relative;
+      margin: 0 auto;
+      width: 170px;
+      font-family: Arial;
```
Comment (pyrmont) [inline spec/visual/templates/layout.erb]: Can this default to the system font? I'm not sure what the keyword for that is in CSS off the top of my head but it should be whatever will produce 'San Francisco' for Apple platforms.

File: spec/visual/templates/layout.erb [hunk 2]
```diff
@@ -38,18 +38,164 @@
...
+        <% end %>
+      </select>
+    </div>
   </footer>
+
+  <script>
+    /*
+     * --------------------------------------------------------------
+     *   Adapted from sample code found at:
+     *     https://www.w3schools.com/howto/howto_custom_select.asp
```
Comment (pyrmont) [inline spec/visual/templates/layout.erb]: Do we really need a custom selector? This isn't a fancy consumer-facing site? A regular HTML5 selector should be enough for our purposes.

File: spec/visual/templates/layout.erb [hunk 3]
```diff
@@ -38,18 +38,164 @@
...
+    #theme-selector {
+      display: inline-block;
+      position: relative;
+      margin: 0 auto;
+      width: 170px;
+      font-family: Arial;
+      font-size: smaller;
+      text-align: left;
+    }
+    #theme-selector select, .select-hide {
```
Comment (pyrmont) [inline spec/visual/templates/layout.erb]: I'd suggest getting rid of all of the CSS used to make the custom selector work. As explained below, this isn't necessary for this kind of site.

**Comment by pyrmont** [comment]
I should have added that, despite my negativity, I do very much like the goal of this PR. Being able to switch themes easily is a good idea.

**Comment by ashmaroli** [comment]
> I believe we should be aiming to keep these pages as simple and straightforward as possible.

Thank you for taking the time for reviewing this @pyrmont. I got carried away and forgot that this is not a consumer-facing page.. :stuck_out_tongue:
 
Nevertheless, I've pushed a minimal version.

**Comment by pyrmont** [comment]
@ashmaroli This is better but honestly, when I ran the PR on my system I didn't realise it was a selection control! And I knew that it was supposed to be one! That's cool in its own way but I think will prevent people from realising that there's a control there.

For mind, a default control is best in this situation. I stripped out all of the CSS and just left the JavaScript (I almost wanted to add in a proper form element so that JavaScript wasn't required but I relented on that—at least for this PR).

I will say that it was great to be able to switch between the themes!

**Comment by ashmaroli** [comment]
I also thought about having a navbar rendered instead of resorting to JavaScript. But that's boring! :stuck_out_tongue:

**Comment by pyrmont** [comment]
@ashmaroli Now you're talking my language :P

**Comment by pyrmont** [comment]
@ashmaroli Do you have anything else you want to add? I'll merge otherwise.

**Comment by ashmaroli** [comment]
> Do you have anything else you want to add?

No. The current state is fine.
Please proceed.

**Pull request merged**